.github/workflows/update-deps.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           ssh-key: ${{ secrets.CI_DEPLOY_KEY }}
 
       - name: Update sample ProjectVersion.txt
-        if: ${{ matrix.unity-prefix == '2019' }}
+        if: ${{ matrix.unity-prefix == '2020' }}
```

Comment:
This is actually irrelevant. Unity 2020 LTS reached its EOL in mid-2023. Specifically, the final patch version was 2020.3.48.